script/github-actions/run-cypress-tests.js

```diff
@@ -14,7 +14,7 @@ const batch = tests
   .join(',');
 
 const status = runCommandSync(
-  `yarn cy:run --reporter cypress-multi-reporters --reporter-options "configFile=config/cypress-reporters.json" --spec '${batch}'`,
+  `yarn cy:run --browser chrome --reporter cypress-multi-reporters --reporter-options "configFile=config/cypress-reporters.json" --spec '${batch}'`,
 );
```

Comment:
One comment here - we were prior using electron and electron runs headless by default.  I don't believe Chrome does the same.  Do we need to specify the `--headless` tag here?

Reply:
Nice catch. [Firefox and Chromium-based browsers do seem to run headed by default](https://docs.cypress.io/guides/guides/command-line#cypress-run-headed). I'll specify that here.

Reply:
I just tried `yarn cy:run --browser chrome` and it doesn't run headlessly.